pontoon/machinery/views.py

```diff
@@ -160,34 +164,34 @@ def microsoft_terminology(request):
 
         obj['locale'] = locale
 
-    sources = client.factory.create('ns0:TranslationSources')
-    sources["TranslationSource"] = ['Terms', 'UiStrings']
-
     payload = {
-        'text': text,
-        'from': 'en-US',
-        'to': locale,
-        'sources': sources,
-        'maxTranslations': 5
+        "uuid": uuid4(),
+        "text": text,
+        "to": locale,
+        "max_result": 5
     }
+    template = get_template('machinery/microsoft_terminology.jinja')
+    payload = template.render(payload)
 
     try:
-        r = client.service.GetTranslations(**payload)
+        r = requests.post(url, data=payload, headers=headers)
         translations = []
-
-        if len(r) != 0:
-            for translation in r.Match:
+        xpath = ".//{http://api.terminology.microsoft.com/terminology}"
+        if r.status_code != 200:
+            return HttpResponseBadRequest('Bad Request: Got {code} status code from service'.format(code=r.status_code))
```

Comment:
I wonder If that information would be useful to the end-user of the pontoon  -> I'd log them to our logs with `.error` level and just do http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status here. That's my personal taste -> the final decision is always on @mathjazz/rest of the folks.